ts/commands/ConfigCommands.ts

```diff
@@ -26,7 +26,7 @@ class ConfigCommands {
           name: 'password',
           type: 'password'
         }, {
-          default: profileConfig.endpoint || 'https://app.netuitive.com',
+          default: profileConfig.endpoint || 'https://app.metricly.com',
```

Comment:
This change is unrelated to this PR, right?

Reply:
yes aesthetics only

Reply:
Ok, you should just push this type of change directly to `develop` instead of lumping it into an unrelated commit and PR.